include/RAJA/util/PluginLinker.hpp

```diff
@@ -0,0 +1,22 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef RAJA_Plugin_Linker_HPP
+#define RAJA_Plugin_Linker_HPP
+
+#include "RAJA/util/RuntimePluginLoader.hpp"
+#include "RAJA/util/KokkosPluginLoader.hpp"
+
+namespace {
+  struct pluginLinker {
+    inline pluginLinker() {
+      (void)RAJA::util::linkRuntimePluginLoader();
+      (void)RAJA::util::linkKokkosPluginLoader();
+    }
+  } pluginLinker;
+}
+#endif
```

Comment:
You'll want to add a newline at the end of this file

Reply:
👍